### ISSUE TITLE
Add Exception Handling for PG_CIRRUS_INSTALLATION_DIRECTORY

### ISSUE THREAD

**Issue by salmanahmed-postgresql** [stormatics/pg_cirrus#86]
As of now pg_cirrus exits when PG_CIRRUS_INSTALLATION_DIRECTORY is not empty. This should be handled through exception handling instead of error by initb command.

**Comment by salmanahmed-postgresql** [comment]
Code executes successfully when PG_CIRRUS_INSTALLATION_DIRECTORY is empty

On primary node
```
TASK [Check if the data directory is empty] *****************************************************************************************************
ok: [PRIMARY]

TASK [Display message and exit if the directory is not empty] ***********************************************************************************
skipping: [PRIMARY]

TASK [Initialize the data directory for the Primary PostgreSQL server] **************************************************************************
changed: [PRIMARY]
```

On standby nodes
```
TASK [Check if the data directory is empty] *****************************************************************************************************
ok: [STANDBY1]
ok: [STANDBY2]

TASK [Display message and exit if the directory is not empty] ***********************************************************************************
skipping: [STANDBY1]
skipping: [STANDBY2]
```

**Comment by salmanahmed-postgresql** [comment]
Tried avoiding the failure message and was successful using the following ansible tasks, but this caused issues with Exception handling used in PR #84. While one playbook stopped execution without considering not empty directory as failure, the other playbooks kept on executing hence not going for this option and considering PG_CIRRUS_INSTALLATION_DIRECTORY not being empty as failure.

```
    - name: Check if data directory is empty
      find:
        paths: "{{ PG_CIRRUS_INSTALLATION_DIRECTORY }}"
        file_type: directory
        patterns: "*"
      register: DIR_CONTENTS

    - name: Display message and exit if directory is not empty
      debug:
        var: DIR_CONTENTS.matched

    - name: Stop playbook execution if directory is not empty
      meta: end_play
      when: DIR_CONTENTS.matched > 0
```

**Comment by salmanahmed-postgresql** [comment]
```DIR_CONTENTS.files | length > 0``` was replaced with ```when: DIR_CONTENTS.matched > 0``` in setup-primary.yml and setup-standby.yml to see if there's any difference, since no difference was observed we're sticking with ```DIR_CONTENTS.files | length > 0```

**Comment by salmanahmed-postgresql** [comment]
This condition ```when: DIR_CONTENTS.files | length == 0``` has been removed from pg_basebackup and initdb tasks in setup-primary.yml and setup-standby.yml playbooks as it is not needed.

**Comment by salmanahmed-postgresql** [comment]
When PG_CIRRUS_INSTALLATION_DIRECTORY is empty the code works as expected

For primary node:
```
TASK [Check if the data directory is empty] *****************************************************************************************************
ok: [PRIMARY]

TASK [Display message and exit if the directory is not empty] ***********************************************************************************
skipping: [PRIMARY]

TASK [Initialize the data directory for the Primary PostgreSQL server] **************************************************************************
changed: [PRIMARY]
```
For standby nodes:
```
TASK [Check if the data directory is empty] *****************************************************************************************************
ok: [STANDBY1]
ok: [STANDBY2]

TASK [Display message and exit if the directory is not empty] ***********************************************************************************
skipping: [STANDBY1]
skipping: [STANDBY2]

TASK [Run pg_basebackup for standby servers] ****************************************************************************************************
changed: [STANDBY1]
changed: [STANDBY2]
```

**Comment by salmanahmed-postgresql** [comment]
Code works as expected when PG_CIRRUS_INSTALLATION_DIRECTORY is not empty

On primary node:

```
TASK [Check if the data directory is empty] *****************************************************************************************************
ok: [PRIMARY]

TASK [Display message and exit if the directory is not empty] ***********************************************************************************
fatal: [PRIMARY]: FAILED! => {"changed": false, "msg": "The data directory '/home/postgres/stormatics/pg_cirrus/data' is not empty."}

PLAY RECAP **************************************************************************************************************************************
PRIMARY                    : ok=13   changed=2    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

```

On standby nodes:
```
TASK [Check if the data directory is empty] *****************************************************************************************************
ok: [STANDBY2]
ok: [STANDBY1]

TASK [Display message and exit if the directory is not empty] ***********************************************************************************
fatal: [STANDBY1]: FAILED! => {"changed": false, "msg": "The data directory '/home/postgres/stormatics/pg_cirrus/data' is not empty."}
fatal: [STANDBY2]: FAILED! => {"changed": false, "msg": "The data directory '/home/postgres/stormatics/pg_cirrus/data' is not empty."}

PLAY RECAP **************************************************************************************************************************************
STANDBY1                   : ok=12   changed=3    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
STANDBY2                   : ok=12   changed=3    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

**Comment by salmanahmed-postgresql** [comment]
This issue has been fixed PR #108 has been created for this issue. After review PR will be merged.

**Comment by salmanahmed-postgresql** [comment]
This issue has been fixed, PR has been merged.
